crates/cli/src/config/mod.rs

```diff
@@ -53,20 +53,32 @@ pub struct Config {
 
 impl Config {
     /// Public constructor function to instantiate a representation of a config file.
-    pub fn new() -> Result<Self, ConfigError> {
+    pub fn new(command_path: &String) -> Result<Self, ConfigError> {
         let mut content = String::new();
 
-        let config_filepaths: [&str; 2] = [
-            "./crates/cli/src/config.toml",
-            "./crates/cli/src/Config.toml",
-        ];
+        match command_path == "" {
```

Comment:
doesn't seem to me there is any circumstance in which it is called with an empty string

Reply:
Would also be taken care of it the command is an option. Still leveraging the default paths

Reply:
The `--config` argument is a string, the default is value is `""` (meaning if they don't input a path it's an empty string) and iterates through the default values.

Reply:
Yeah was under the impression `default_value = ""` meant if you don't specify a path it won't have an error, but see this is not the case. Will fix